### ISSUE TITLE
IVT using firmware 04.07.07 lacks dhw paths

### ISSUE THREAD

**Issue by hansliss** [bosch-thermostat/bosch-thermostat-client-python#22]
My heatpump (IVT Geo 312c) previously (a couple of days ago) had a lot of stuff under /dhwCircuits. Now there's nothing - dhw1 disappeared, presumably after it updated to 04.07.07. Is it just me? Did the domestic hot water info move to another path?

[
    {
        "id": "/dhwCircuits",
        "type": "refEnum",
        "references": []
    }
],

**Comment by pszafer** [comment]
Can you try to restart your installation? AFAIK it's working for other people

**Comment by hansliss** [comment]
I confirmed the same thing with other software as well. It was like that for at least an hour or two while I was troubleshooting. 

But never mind, it just reappeared, about five minutes after I restarted the heatpump. It wasn't there immediately after the restart, looking just like in the original post, but then it appeared - the entire subtree below /dhwCircuits.

Maybe the network module lost the connection to the dhw subsystem for some reason.

Sorry for any confusion, but perhaps it might be interesting for you to know that this can happen.
